plugins/lookup/supported_roles.py

```diff
@@ -48,13 +50,17 @@
         'setup_repmgr',
         'manage_dbpatches',
         'manage_efm',
+        'setup_etcd',
+        'setup_patroni',
     ],
     'pemserver': [
         'setup_repo',
         'install_dbserver',
         'init_dbserver',
         'manage_dbserver',
         'setup_pemserver',
+        'setup_etcd',
```

Comment:
should the `setup_etcd` role be included in the default `supported_roles` or setup like in lines 202-210 now that it is set in the inventory? In the `playbook-examples/bdr/inventory-bdr.yml` there are cases where `etcd` is not set in all hostvars and having it defined here may have it run in instances where we do not want it to.

Reply:
Yeah. This need to be fixed if we want to cover the use case of PGD. @jt-edb, Can you please check the above feedback and share your comments on how we want to cover the topics for PGD? It may not be needed now. However, in the future, we need to work on that.

Reply:
Fixed